draft-ietf-suit-report.md

```diff
@@ -652,8 +652,9 @@ be taken to ensure that this does not leak information and that the
 principle of least privilege is respected. For example, in an EAT-based
 attestation workflow, the Verifier often will not need the full SUIT
 Report. Similarly, the Relying Party may also not need the SUIT\_Report.
-In this case, the SUIT\_Report MUST be encrypted even if the EAT token
-that contains it is also encrypted.
+To enable the principle of least privilege in this and similar
+scenarios, the SUIT\_Report should be independently encrypted even if
+the EAT token or encrypted transport that contains it is also encrypted.
```

Comment:
```suggestion
To enable the principle of least privilege in this and similar
scenarios, the SUIT\_Report SHOULD be independently encrypted even if
the EAT and/or the transport that contains it is also encrypted.
```

Reply:
Does this require normative language? I don't think it does. It's a guideline not a protocol action.